bigquery/google/cloud/bigquery/job.py

```diff
@@ -1033,7 +1069,7 @@ def from_api_repr(cls, resource):
             An extract job configuration in the same representation as is
             returned from the API.
 
-        :rtype: :class:`google.cloud.bigquery.job.ExtractJobConfig`
+        :rtype: :class:`google.cloud.bigquery.job.CopyJobConfig`
```

Comment:
Looks like this method belongs to ExtractJobConfig. Probably meant to copy this to the CopyJobConfig class? We may wish to make this part of a shared base class.

Reply:
Fixed.